CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v4.21.1 (unreleased)
+
+* Add `show_expired` argument to `Conjur::Variable#value` to allow
+  retrieval of values of expired variables.
+* Fix `Conjur::API#variable_expirations` so it returns an array of
```

Comment:
Strictly speaking, this is a breaking API change. But, I suspect no one will notice it except me.